src/clojurians_log/routes.clj

```diff
@@ -132,6 +132,15 @@
       views/about
       response/render))
 
+(defn user-profile-route  [{:keys [endpoint] :as request}]
+    (let [db (db-from-endpoint endpoint)]
+      (-> request
+          make-context
+          (assoc :data/username 
```

Comment:
I would expect to find a `username` under `:data/username`, can we call this `:data/user-profile`?

Reply:
yes, you are right.
I will change this.